Create ui-c8y-1021-19-0-added-docs-information-in-the-modal-of-smart-group-creation.md (#2707)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-19-0-added-docs-information-in-the-modal-of-smart-group-creation.md

---
date: ""

> title: Added docs information in the modal of smart group creation.

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add documentation link to smart group creation modal
```

> title: Added docs information in the modal of smart group creation.

Reviewer: ```suggestion
title: Helper link with additional documentation about Smart Groups available in the smart group creation modal.
```

> title: Helper link with additional documentation about Smart Groups available in the smart group creation modal.

Reviewer 2: ```suggestion
title: Added link to user documentation in the smart group creation dialog
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4130
version: 1021.19.0
---

> Added docs information in the modal of smart group creation.

Author: The Claude AI suggested to use the following body:
```suggestion
To make it easier for users to find relevant information when creating smart groups, a link to the smart group documentation has been added to the smart group creation modal. This allows users to quickly access helpful information and examples without having to search for the documentation separately. The change is available in the UI whenever a user creates a new smart group and opens the smart group creation modal.
```

> Added docs information in the modal of smart group creation.

Reviewer: ```suggestion
To make it easier for users when creating smart groups, a link to the smart group documentation has been added to the smart group creation modal. This allows users to quickly access helpful information.
```

> To support users when creating smart groups, a link to the user documentation has been added to the smart group creation modal. This allows users to quickly access helpful information.

Reviewer 2: ```suggestion
To support users when creating smart groups, a link to the user documentation on smart groups has been added to the smart group creation dialog. This allows users to quickly access helpful information.
```

> To support users when creating smart groups, a link to the smart group documentation has been added to the smart group creation modal. This allows users to quickly access helpful information.

Reviewer 2: ```suggestion
To support users when creating smart groups, a link to the user documentation has been added to the smart group creation modal. This allows users to quickly access helpful information.
```

> To make it easier for users when creating smart groups, a link to the smart group documentation has been added to the smart group creation modal. This allows users to quickly access helpful information.

Reviewer 2: ```suggestion
To support users when creating smart groups, a link to the smart group documentation has been added to the smart group creation modal. This allows users to quickly access helpful information.
```
